binderhub/templates/page.html

```diff
@@ -42,7 +42,7 @@
   {% block footer %}
   <div class="container">
     <div class="row text-center">
-      <h3>questions?<br />join the <a href="https://discourse.jupyter.org/c/binder">discussion</a>, read the <a href="https://mybinder.readthedocs.io/en/latest/">docs</a>, see the <a href="https://github.com/jupyterhub/binderhub">code</a></h3>
+      <h3>questions?<br />join the <a lang ="en"href="https://discourse.jupyter.org/c/binder">discussion</a>, read the <a href="https://mybinder.readthedocs.io/en/latest/">docs</a>, see the <a href="https://github.com/jupyterhub/binderhub">code</a></h3>
```

Comment:
This line of code seems to be what the PR was meant to update. The PR currently has a lot of other changes, probably caused by issues trying to resolve a git merge conflict.

I think the easiest approach to resolving this is to recreate the PR from scratch adding this change. Make sure to put a space before and after `lang="en"` also!

Reply:
I'll go for a close on this PR for now, it can be reopened as well if you wish!